collection-scripts/gather_metal3

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Resource List
+resources=()
+
+# Metal3 related Resources
+resources+=(ns/openshift-machine-api ns/openshift-machine-config-operator)
```

Comment:
Don't we already collect this with related resources and cluster operators?

Reply:
My understanding was that we need to collect this. @etingof please confirm

Reply:
@sferich888's comment makes sense to me. But I'd defer this to @dhellmann to be sure.